contrib/scalapblib/src/ScalaPBWorker.scala

```diff
@@ -108,7 +106,7 @@ class ScalaPBWorker extends AutoCloseable {
 
 trait ScalaPBWorkerApi {
   def compileScalaPB(
-      root: File,
```

Comment:
I suggest to keep an overloaded (maybe deprecated) method with the old signature (it can simply forward to the new implementation though), to keep binary compatibility and avoid breaking someones workflow. You never know, who is already using or integrating this module.